consul_kv_migrator.py

```diff
@@ -0,0 +1,97 @@
+from consul_kv import Connection
+import json
+import ast
+from prettytable import PrettyTable
+import getopt
+import sys
+
+argv = sys.argv[1:]
+
+def _emptyCheckerFunc(argv):
+  if not argv:
+    print("use -h tag to get more information about the utility")
+    exit()
+
+_emptyCheckerFunc(argv)
+
+try:
+  opts, args = getopt.getopt(argv, "s:d:o:h")
+
+except getopt.GetoptError:
+  print("USAGE:")
+  print("-h:  To get more information about the script options")
+  print("FORMAT: python script.py -h")
+  print("")
+  exit()
+
+
+for opt, arg in opts:
+
+  if opt in ['-s']:
+    _sourceConsulLink = arg
+  if opt in ['-d']:
+    _destinationConsulLink = arg
+  if opt in ['-o']:
+    _outputType = arg
+  elif opt in ['-h']:
+    print("Usage: python <NAME-OF-SCRIPT>.py ... [ -<OPTIONS> <VALUES>]")
+    print("")
```

Comment:
you dont need to use this line.. if you want to put nextline , you can use \n in line 37 ..

Reply:
Done with changes